app/models/pulitzer/post_type_content_element_type.rb

```diff
@@ -4,21 +4,72 @@ class PostTypeContentElementType < ActiveRecord::Base
     
     belongs_to :post_type
     belongs_to :content_element_type
-    has_one :content_element
+    has_many :styles
+    
     before_save :handle_sort_order
 
-    delegate :type, :image_type?, to: :content_element_type
+    delegate :type, :image_type?, :has_styles?, to: :content_element_type
 
     default_scope { order(sort_order: :asc) }
 
     validates :label, presence: true
 
+    def self.any_clickable
+      return @any if @any.present?
+      @any = OpenStruct.new
```

Comment:
I think the way I wrote it is misleading, I'll rewrite it to be clear that these are immutable class constants.